### PR TITLE
Fix command for running custom storageEngine on mongodb

### PR DESCRIPTION
**Pull request by josegonzalez**

### CODE REVIEW

File: mongo/content.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ See the [official docs](http://docs.mongodb.org/manual/) for infomation on using
 Just add the `--storageEngine` argument if you want to use the WiredTiger storage engine in MongoDB 3.0 and above without making a config file. Be sure to check the [docs](http://docs.mongodb.org/manual/release-notes/3.0-upgrade/#change-storage-engine-to-wiredtiger) on how to upgrade from older versions.
 
 ```console
-$ docker run --name some-mongo -d mongo --storageEngine=wiredTiger
+$ docker run --name some-mongo -d mongod --storageEngine wiredTiger
```
Comment (tianon) [inline mongo/content.md]: Uh, I don't think this does what you intended:
```console
$ docker run --name some-mongo -d mongod --storageEngine wiredTiger
Unable to find image 'mongod:latest' locally
Pulling repository docker.io/library/mongod
Error: image library/mongod:latest not found
```

Comment (josegonzalez) [inline mongo/content.md]: Oh heh oops, should be `mongo mongod` :|

Comment (tianon) [inline mongo/content.md]: On the other hand, what's already there appears to work great here:
```console
$ docker run --name some-mongo -d mongo --storageEngine=wiredTiger
Unable to find image 'mongo:latest' locally
latest: Pulling from library/mongo

4c8cbfd2973e: Already exists 
60c52dbe9d91: Already exists 
9eddd1a555fc: Pull complete 
675abd82e9c0: Pull complete 
0e1f126ca2a3: Pull complete 
403a86009909: Pull complete 
71935c220489: Pull complete 
e18ca47dafd5: Pull complete 
a96c4dc8c978: Pull complete 
78b590e9add0: Pull complete 
6da089a621ff: Pull complete 
9f673fbd6c0b: Pull complete 
0fed7055870c: Pull complete 
b42e035fdac8: Pull complete 
7f3f71b42319: Pull complete 
6360e8024080: Pull complete 
c6f67f622b2a: Pull complete 
library/mongo:latest: The image you are pulling has been verified. Important: image verification is a tech preview feature and should not be relied on to provide security.
Digest: sha256:9e27cd0774f3d2e67f45db459b3d7838641a437a94900665eb9ca6a2f8b59202
Status: Downloaded newer image for mongo:latest
828042f2458b28554ea0ddf77c86742d6293468a7b657749a8288c4b66482b1f
$ docker logs --tail=20 some-mongo
2015-08-24T17:00:54.337+0000 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=15G,session_max=20000,eviction=(threads_max=4),statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),checkpoint=(wait=60,log_size=2GB),statistics_log=(wait=0),
2015-08-24T17:00:55.176+0000 I CONTROL  [initandlisten] MongoDB starting : pid=1 port=27017 dbpath=/data/db 64-bit host=828042f2458b
2015-08-24T17:00:55.176+0000 I CONTROL  [initandlisten] db version v3.0.5
2015-08-24T17:00:55.176+0000 I CONTROL  [initandlisten] git version: 8bc4ae20708dbb493cb09338d9e7be6698e4a3a3
2015-08-24T17:00:55.176+0000 I CONTROL  [initandlisten] build info: Linux ip-10-47-180-245 3.2.0-4-amd64 #1 SMP Debian 3.2.46-1 x86_64 BOOST_LIB_VERSION=1_49
2015-08-24T17:00:55.176+0000 I CONTROL  [initandlisten] allocator: tcmalloc
2015-08-24T17:00:55.176+0000 I CONTROL  [initandlisten] options: { storage: { engine: "wiredTiger" } }
2015-08-24T17:00:55.411+0000 I NETWORK  [initandlisten] waiting for connections on port 27017
```

**Comment by josegonzalez** [comment]
SORRY!

File: mongo/content.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ See the [official docs](http://docs.mongodb.org/manual/) for infomation on using
 Just add the `--storageEngine` argument if you want to use the WiredTiger storage engine in MongoDB 3.0 and above without making a config file. Be sure to check the [docs](http://docs.mongodb.org/manual/release-notes/3.0-upgrade/#change-storage-engine-to-wiredtiger) on how to upgrade from older versions.
 
 ```console
-$ docker run --name some-mongo -d mongo --storageEngine=wiredTiger
+$ docker run --name some-mongo -d mongo mongod --storageEngine wiredTiger
```
Comment (tianon) [inline mongo/content.md]: I'm still confused why this is necessary though, given the following block in the entrypoint script, which will see the leading `-` and prepend `mongod` automatically: https://github.com/docker-library/mongo/blob/c664ab653154aa4068a46f57ceeacd7c5cda1ae4/3.0/docker-entrypoint.sh#L4-L6

Comment (josegonzalez) [inline mongo/content.md]: Gotcha. Well originally I was just fixing the `=` to be a space, and then it appeared that said change didn't work without also adding the binary. Seems like it does indeed work, so I'll remove the binary. My bad.

**Comment by yosifkit** [comment]
Works with or without equals sign, but LGTM.

**Comment by josegonzalez** [comment]
@yosifkit weird, it actually bombed out for me... Either way, this way appears to work.

**Comment by tianon** [comment]
Ok, seems fair. :+1:

LGTM

**Comment by josegonzalez** [comment]
Thanks :)

**Pull request merged**